### PR TITLE
Bumped Symfony version

### PR DESCRIPTION
**Pull request by DonCallisto**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,14 +12,12 @@ env:
         - SYMFONY_PHPUNIT_DIR=$HOME/.phpunit-bridge
         - SYMFONY_PHPUNIT_REMOVE='' # don't remove prophecy
 
-php: [7.2, 7.1, 7.0, 5.6, 5.5, 5.4]
+php: [7.2, 7.1]
```
Comment (stof) [inline .travis.yml]: Dropping support for older PHP version in a PR described as bumping Symfony version ? That's not good description of what the PR does.

Comment (DonCallisto) [reply .travis.yml]: You're right but without those drops the CI won't go green.
After this https://github.com/Incenteev/ParameterHandler/pull/129 is merged, this change won't be there anymore.

**Comment by stof** [comment]
I'm closing this PR, because it is not at all about supporting new Symfony versions, but about dropping some PHP versions, which is not necessary for now, and is not what this PR claims to do.

**Comment by DonCallisto** [comment]
@stof have you even read my comment?

**Comment by stof** [comment]
#132 has added support for Symfony 5, without dropping anything of the current support.

**Comment by DonCallisto** [comment]
@stof So? what's the point here? Is it a comparison? 
I've explicitly stated the motivations behind this choice.
If you don't want to drop the older PHP versions let's state clearly it the related PR (https://github.com/Incenteev/ParameterHandler/pull/129) not here.
If the related PR was closed (and it's was not) and the red CI is acceptable from your POV I would have go with your indications.
Behaving like you have done here is a no-go from my standpoint. Yes you can act as you prefer and I can only imagine how much OSS work you have (thanks, by the way!) but this kind of miscommunication is what tend to keep some people out of OSS contribution.

**Comment by DonCallisto** [comment]
Don’t get me wrong, I’m not complaining about having this PR closed (what you stayed in #129 is perfectly understandable), just sayin’ it would have been better to explain what   supplementare work could have been done in order to make this PR acceptable.
Closing with te motivation of php version bump (that’s not the main focus of the PR) is totally unrelated as the bump itself.

**Comment by xabbuh** [comment]
@DonCallisto Can you explain what you think is missing from this PR after was merged?

**Comment by DonCallisto** [comment]
I was not arguing against having close this PR.
However just bumping the minors still supported but as stof said is something you don’t want to do, and rightfully so after his explaination.

**Pull request closed without merging**